tests/integration_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -141,7 +149,14 @@ def _dataset_tables(self, dataset_id):
         tables = bq_utils.list_tables(dataset_id)
         return [table['tableReference']['tableId'] for table in tables]
 
-    def test_union_ehr(self):
+    @mock.patch('bq_utils.get_hpo_info')
+    def test_ehr_main_exclude_hpo_id(self, mock_hpo_info):
+        mock_hpo_info.return_value = [{
+            'hpo_id': hpo_id
+        } for hpo_id in self.hpo_ids]
```

Comment:
This looks like it is not testing anything. You can remove it.

Reply:
done